app/javascript/components/recipes/new/selected-ingredients.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="flex w-80 flex-col">
+    <span class="flex content-center justify-center">Ingredientes seleccionados</span>
+    <div 
+    class="flex p-4 grid grid-cols-2 items-center justify-between "
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 6 spaces but found 4 spaces    vue/html-indent